helix-core/src/main/java/org/apache/helix/task/TaskStateModelFactory.java

```diff
@@ -44,11 +49,11 @@
   private final ScheduledExecutorService _taskExecutor;
   private final ScheduledExecutorService _timerTaskExecutor;
   private ThreadPoolExecutorMonitor _monitor;
-  public final static int TASK_THREADPOOL_SIZE = 40;
+  public final static int DEFAULT_TASK_THREAD_POOL_SIZE = 40;
```

Comment:
Could we potentially move this constant to TaskConstants? Do you think that would be a good idea?

Reply:
Since it's only used in this class, would it be better to have it here? And make it private?

Reply:
I realized that this is actually incorrect. I'll move it to TaskConstants.